shapiq/games/imputer/base.py

```diff
@@ -19,30 +19,67 @@ class Imputer(Game):
             with shape ``(n_samples, n_features)``.
         categorical_features: A list of indices of the categorical features in the background data.
         random_state: The random state to use for sampling. Defaults to ``None``.
+
+    Attributes:
+        n_features: The number of features in the data (equals the number of players in the game).
+        data: The background data to use for the imputer.
+        model: The model to impute missing values for as a callable function.
+        sample_size: The number of samples to draw from the background data.
     """
 
     @abstractmethod
     def __init__(
         self,
         model,
         data: np.ndarray,
+        x: Optional[np.ndarray] = None,
+        sample_size: int = 100,
```

Comment:
It would be nice to add information about x and sample_size in the docs of __init__.
Could just use the identical comments for BaselineImputer

Reply:
thanks overlooked this!